src/core/IronPython/Runtime/Bytes.cs

```diff
@@ -1241,10 +1241,13 @@ Expression IExpressionSerializable.CreateExpression() {
 
         #region IBufferProtocol Support
 
-        IPythonBuffer IBufferProtocol.GetBuffer(BufferFlags flags) {
-            if (flags.HasFlag(BufferFlags.Writable))
-                throw PythonOps.BufferError("Object is not writable.");
-
+        IPythonBuffer? IBufferProtocol.GetBuffer(BufferFlags flags, bool throwOnError) {
+            if (flags.HasFlag(BufferFlags.Writable)) {
+                if (throwOnError) {
+                    throw PythonOps.TypeError("bytes object is not writable");
```

Comment:
Why a `TypeError` on this one?

Reply:
Unintentional. VS Code trying to be helpful…